.travis.yml

```diff
@@ -12,8 +12,24 @@ before_script:
 
 script:
     - cd $TRAVIS_BUILD_DIR
-    - echo "TODO need to build testing with samples"
-
+    - curl https://api.github.com/repos/vitasdk/autobuilds/releases  | grep browser_download_url | grep linux | head -n 1 | cut -d '"' -f 4 | xargs wget
```

Comment:
yes. i saw before. it's just my curious thing(and also little things).
curl and wget are very similar application, so we can reduce this complex

options:
1. `curl` change to `wget -qO-`
2. `wget` change to `curl` or `curl -L`

both cases can join next tar extract command.